build/iptables-wrapper-installer.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/sh
```

Comment:
@mazocode Please move /build/iptables-wrapper-installer.sh to /scripts/iptables-wrapper-installer.sh

Reply:
ok, I've changed the path.